frontend/src/component/Workspace/ContentItemHeader.jsx

```diff
@@ -20,6 +21,16 @@ export const FileItemHeader = props => {
           </div>
         </div>
       )}
+      {props.showLastEdit && !props.showSearchDetails && (
+        <div className='content__header__modif'>
```

Comment:
I think it will be better to avoid abbreviations and use the same words everywhere

```suggestion
      {props.showLastModification && !props.showSearchDetails && (
        <div className='content__header__modification'>
```